rules/autoconsent/consent-manager.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "consent-manager",
+    "cosmetic": false,
+    "runContext": {
+        "main": true,
+        "frame": false
+    },
+    "prehideSelectors": [],
+    "detectCmp": [
+        {
+            "exists": "[data-cy=\"consent-manager-banner\"]"
```

Comment:
@sammacbeth I believe data-cy attributes are used by the Cypress test framework, so it's probably a very site-specific thing. Should we add a urlPattern?